python/pyspark/ml/tests.py

```diff
@@ -1075,7 +1076,8 @@ def test_linear_regression_summary(self):
         pValues = s.pValues
         self.assertTrue(isinstance(pValues, list) and isinstance(pValues[0], float))
         # test evaluation (with training dataset) produces a summary with same values
-        # one check is enough to verify a summary is returned, Scala version runs full test
+        # one check is enough to verify a summary is returned
+        # The child class LinearRegressionTrainingSummary runs full test
```

Comment:
I think this is not because Scala version runs full test. Even Scala version runs full test, we still need the function call test.
If a child class have done the function call test, we don't need to test parent class again.